src/pg/test/e2e.test.js

```diff
@@ -439,6 +439,81 @@ describe('pg_e2e', () => {
     expect(res2).toEqual(null);
   });
 
+  describe('val', () => {
+    const id = uuid();
+    // beforeAll(async () => {
+    //   const result = await store.write('users', [
+    //     {
+    //       $key: id,
+    //       $put: true,
+    //       name: 'A',
+    //       email: 'a',
+    //       settings: {
+    //         foo: {
+    //           bar: 'incorrect',
+    //           baz: 'correct'
+    //         }
+    //       }
+    //     },
+    //   ]);
+    // });
+    // afterAll(async () => {
+    //   await store.write(['users', id], null)
+    // })
```

Comment:
As soon as I try to comment this out and move the code from below to this, it fails with `pg.nothing_written` error. Not sure what I m doing wrong.

Reply:
![image](https://user-images.githubusercontent.com/10926994/225268304-db7afc8d-5e68-432b-b3e4-3951d6612553.png)